client/src/pages/GetAllTasks.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const GetAllTasks = () => {
+  return (
+    <div>
+      
+    </div>
+  )
```

Comment:
Since this is just empty, I guess you can also return null, it should work.

The issue with empty divs is that it makes the UI more complex for no reason.